pom.xml

```diff
@@ -133,7 +133,6 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.compiler.fork>true</maven.compiler.fork>
-        <maven.compiler.verbose>true</maven.compiler.verbose>
```

Comment:
Also just wondering about removing this? I think I added it when we were having out-of-memory problems with javac but I'm not sure.